pkgs/development/tools/misc/cc-tool/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv, fetchFromGitHub,
+  autoconf, automake, boost, libusb1, libtool, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  name = "cc-tool-${version}";
+  version = "unstable-2020-05-19";
+  
+  src = fetchFromGitHub {
+    owner = "dashesy";
+    repo = "cc-tool";
+    rev = "19e707eafaaddee8b996ad27a9f3e1aafcb900d2";
+    sha256 = "1f78j498fdd36xbci57jkgh25gq14g3b6xmp76imdpar0jkpyljv";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ libusb1 boost autoconf libtool automake ];
+
+  configurePhase = ''
+    ./bootstrap
+    ./configure prefix=$out
+  '';
+
+  installPhase = ''
+    mkdir -p $out
+    make install
+    install -D udev/90-cc-debugger.rules $out/lib/udev/rules.d/90-cc-debugger.rules
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Command line tool for the Texas Instruments CC Debugger";
+    longDescription = ''
+      cc-tool provides support for Texas Instruments CC Debugger
+    '';
+    homepage = "";
```

If the project doesn't have a home page, just use its GitHub page:

```suggestion
    homepage = "https://github.com/dashesy/cc-tool";
```